features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -150,6 +150,12 @@
 #ifdef MBED_CONF_LWIP_PBUF_POOL_BUFSIZE
 #undef PBUF_POOL_BUFSIZE
 #define PBUF_POOL_BUFSIZE           MBED_CONF_LWIP_PBUF_POOL_BUFSIZE
+#else
```

Comment:
Could also use `#elif` here. Not sure if it's better or worse, but consider it.

Reply:
Duly noted.